mcp_server.py

```
@@ -227,6 +227,83 @@ async def list_devices() -> Dict[str, Any]:
         raise Exception(f"Failed to list devices: {str(e)}")
 
 
+async def add_device(
+    hostname: str,
+    device_type: str = "server",
+    description: Optional[str] = None,
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)